docs/make.jl

```diff
@@ -17,15 +19,15 @@ makedocs(
         "GDAL Datasets" => "datasets.md",
         "Feature Data" => "features.md",
         "Raster Data" => "rasters.md",
+        "Working with Images" => "images.md",
         "Tables Interface" => "tables.md",
         "Geometric Operations" => "geometries.md",
         "Spatial Projections" => "projections.md",
         # "Working with Spatialite" => "spatialite.md",
```

Comment:
Maybe remove this commented out section? Edit: uncomment this, there seems to be a spatialite section.

Reply:
Added a note for it in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/2fd5aeec811f4abd159de4c4a818407914e4b9eb, thanks!